packages/modules/src/modules/banners/choiceCardsBanner/ChoiceCardsBanner.stories.tsx

```diff
@@ -29,6 +29,7 @@ const Template: Story<ChoiceCardStoryProps> = (props: ChoiceCardStoryProps) =>
             {...props}
             backgroundColor={props.backgroundColor}
             headingColor={props.headingColor}
+            borderTop={props.borderTop}
```

Comment:
It feels a bit wrong to use `headingColor`  for the border's colour? If these were to be different in the future we'd need to refactor.  

Instead of passing a `boolean` for `borderTop` and reusing `headingColor` for the border's colour could you pass the hexcode as `borderTopColor`, and could `borderTopColor` be optional? And we only define a `border-top` container override if `borderTopColor` has been defined?

Reply:
Yes that would be much better, have done a commit 'borderTop optional colour passed through' with the optional colour approach. Then have done further commit 'serialized style approach' with Maria serialised styles recommendation. Unsure which was preferable so finished on the later.